lib/sles4sap/sap_deployment_automation_framework/configure_tfvars.pm

```diff
@@ -144,12 +146,18 @@ sub set_image_parameters {
     my @variable_names = qw(SDAF_IMAGE_PUBLISHER SDAF_IMAGE_OFFER SDAF_IMAGE_SKU SDAF_IMAGE_VERSION);
     # This maps a variable name from array @variable names to value from delimited 'PUBLIC_CLOUD_IMAGE_ID' parameter
     # Order is important here
-    @params{@variable_names} = split(':', get_required_var('PUBLIC_CLOUD_IMAGE_ID'));
 
     # Add all remaining parameters with static values
     $params{SDAF_IMAGE_OS_TYPE} = 'LINUX';    # this can be modified in case of non linux images
-    $params{SDAF_SOURCE_IMAGE_ID} = '';    # for supplying uploaded image - not implemented yet
-    $params{SDAF_IMAGE_TYPE} = 'marketplace';
+    if (is_azure() && get_var('PUBLIC_CLOUD_IMAGE_LOCATION')) {
```

Comment:
I think you can remove the `is_azure` function completely. SDAF can/will be run only on Azure.

Reply:
Sure, I will remove is_azure.